### ISSUE TITLE
Template failing to load for Debian Wheezy systems using Hiera

### ISSUE THREAD

**Issue by dannykansas** [maverixbio/puppet-stackdriver-agent#22]
I'm using Hiera, and receiving the following error (using a Hiera backend, puppetmaster version 3.7.1, puppet agent version 3.6.2):

```
Error: Could not retrieve catalog from remote server: Error 400 on SERVER: Could not find template 'stackdriver/Linux//etc/default/stackdriver-agent.erb' at /etc/puppet/environments/live/modules/stackdriver/manifests/config/debian.pp:30 on node redacted.redacted.com
Warning: Not using cache on failed catalog
Error: Could not retrieve catalog; skipping run
```

As you can see, it appears that the template path referenced in manifests/config/debian.pp on line 30 is correctly interpolating variables, but not placing "templates" in the path as would be expected (i.e. "stackdriver/templates/Linux/etc/default/stackdriver-agent.erb").

Additionally, there's an extra forward slash in the path. To test whether this was the issue, I removed a forward slash between ${::kernel} and ${sysconfig} in line 30 of the same file. Same error, just minus one of the forward slashes (as expected).

Any clue what might be going on here?

EDIT: I am unable to test this on other systems, so the mention of "Debian Wheezy" is not indicative of success using my setup with a different operating system. Just another datapoint.

**Comment by dannykansas** [comment]
Turns out that the issue was only on the v1.1 tag (currently the active version on Puppet Forge).

The offending file was a misname of the [Debian template for /etc/default/stackdriver-agent](https://github.com/maverixbio/puppet-stackdriver-agent/blob/v1.1/templates/Linux/etc/default/stackdriver.erb). 

As you can see in the debian [config manifest, line 19](https://github.com/maverixbio/puppet-stackdriver-agent/blob/v1.1/manifests/config/debian.pp#L19) of that same tag, the template name will never be found due to the setting of `$sysconfig`.

@maverixbiomicsm I've forked this and tagged at the most recent commit for my own purposes, but I'd like to be sure this gets back to the community. I don't want to submit an alternate module to the Forge if this is being actively maintained and tagged, so I'll hold off for a bit.

**Comment by dstockman** [comment]
@dannykansas I'm a bit slammed at the moment but can get this sorted out in a bit. The module is being maintained.. I'll get an updated version on the forge soon. Thx.

**Comment by dstockman** [comment]
Just a heads up that I released 1.2.0 on the Forge and the repo has moved to my github account from maverixbio.
